go/worker/registration/registration.go

```diff
@@ -228,12 +228,20 @@ func (r *Registration) registerNode(epoch epochtime.EpochTime) error {
 }
 
 func (r *Registration) consensusValidatorHook(n *node.Node) error {
+	var consensusAddrs []node.ConsensusAddress
 	addrs, err := r.consensus.GetAddresses()
 	if err != nil {
 		return errors.Wrap(err, "worker/registration: failed to get consensus validator addresses")
 	}
+	id := r.consensus.ConsensusKey()
+	for _, addr := range addrs {
```

Comment:
Why not make `GetAddresses` return consensus addresses instead of needing to merge them here?

Reply:
Hmm... I don't know why I didn't change the consensus API to have:

```go
// GetAddresses returns the consensus backend addresses.
GetAddresses() ([]node.ConsensusAddress, error)
```

Sure, I can change it.